pkgs/development/libraries/libxcrypt/default.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, libtool, pkg-config }:
```

Following my personal taste comment:

```suggestion
{ stdenv
 ,fetchFromGitHub
 ,autoreconfHook
 ,libtool
 ,pkg-config
}:
```